client/state/data-layer/wpcom/sites/scan/index.js

```diff
@@ -115,11 +117,20 @@ const onFetchStatusSuccess = ( action, scan ) => ( dispatch ) => {
 		( threat ) => threat.fixerStatus === 'in_progress'
 	);
 	if ( action.pooling && scan.state === 'idle' && threatsFixedInProgress.length > 0 ) {
```

Comment:
Would be helpful to get another test on this PR:
* added guards to prevent potential UI rerendering infinitely
* let it silently retry in the background so the user doesn’t see the UI flickering while the code is retrying the API
You can test it out by changing this like to `if ( true )`